README.md

```diff
@@ -46,31 +46,31 @@ Please make sure to use the correct system configuration file that corresponds t
 ```bash
 cloudai\
     --mode install\
-    --system_config_path conf/v0.6/general/system/example_slurm_cluster.toml
+    --system-config conf/v0.6/general/system/example_slurm_cluster.toml
```

Comment:
- We need to update CloudAIX arguments and the README for consistency.
- We should inform users that the usage will be updated (#cloudai-users).

Reply:
I'm not sure we should mirror this project and extensions. It might be even better to keep them as different as possible, because they are different :)

Anyway, I'm OK to update it there too, no problem. Once merged, we can create a new pre-release and update our users. Plus prepare update for Extensions.

Reply:
The intuition to remove _path makes sense. Given that's the case, we should do the same fix in CloudAIX.